tests/rhosak/kas_with_instance.spec.ts

```diff
@@ -219,27 +216,55 @@ test('check Topic does not exist and create and delete', async ({ page }) => {
   // expecting not to find topic row
   await expect(page.locator('td', { hasText: `${testTopicName}` })).toHaveCount(0);
 
-  await createKafkaTopic(page, testTopicName);
+  await createKafkaTopic(page, testTopicName, true);
   await deleteKafkaTopic(page, testTopicName);
 });
 
 // test_4kafka.py test_kafka_try_create_topic_with_same_name
 test('test kafka try create topic with same name', async ({ page }) => {
-  await waitForKafkaReady(page, testInstanceName);
   await page.locator('a', { hasText: `${testInstanceName}` }).click();
   await page.locator('button[aria-label="Topics"]').click();
-  await createKafkaTopic(page, testTopicName);
+  await createKafkaTopic(page, testTopicName, true);
   await expect(page.locator('tr', { hasText: `${testTopicName}` })).toHaveCount(1);
   await page.locator('button', { hasText: 'Create topic' }).click();
   await page.getByPlaceholder('Enter topic name').fill(testTopicName);
   await page.locator('button', { hasText: 'Next' }).click();
   await expect(page.getByText(`${testTopicName}` + ' already exists. Try a different name')).toBeVisible();
 });
 
+test('create Topic with properties different than default', async ({ page }) => {
+  await navigateToKafkaTopicsList(page, testInstanceName);
+  await createKafkaTopic(page, testTopicName, false);
+
+  // Checking phase
+  await navigateToProperties(page, testInstanceName, testTopicName);
+
+  await expect(await page.getByLabel('Partitions').getAttribute('value')).not.toBe(1);
+
+  const rt = await page
+    .locator(
+      'section[role="group"]:has-text("Core configurationBefore deploying your topic, we recommend entering all core co")'
+    )
+    .getByLabel('Retention time')
+    .getAttribute('value');
+  await expect(rt).not.toMatch('604800000 ms (7 days)');
+
+  const rs = await page.getByLabel('Retention size').getAttribute('value');
+  await expect(rs).not.toMatch('Unlimited');
+
+  const cp = await page.getByLabel('Cleanup policy').getAttribute('value');
+  await expect(cp).not.toMatch('delete');
+
+  // Topic CleanUp
+  await page.locator('button', { hasText: 'Delete topic' }).click();
+  await page.getByLabel('Type DELETE to confirm:').fill('DELETE');
+  await page.locator('footer').locator('button', { hasText: 'Delete' }).click();
+});
```

Comment:
Why are you not testing whether the properties you have set are there and testing just if properties are other than default?

Reply:
Plus duplicating code for topic deletion, instead of using `deleteKafkaTopic`.

Reply:
this is because at the bottom of the page there is a Delete Topic button that can be tested on the go

Reply:
I tested regression and with True parameter these tests work also if you don't specify true or false that is an error 
<img width="967" alt="image" src="https://user-images.githubusercontent.com/100594859/217249685-20b74652-917b-4892-ae2a-b4c212cf3792.png">
translated: 3 arguments were expected but found 2

Reply:
Yeah, but you did not set a default value to `defaultProperties` that would be used should you not pass anything as the 3rd argument.